Automattic/sensei
# Add content drip upgrade metabox to edit lesson screen

Fixes #4755

### Changes proposed in this Pull Request

* Add content drip upgrade metabox to edit lesson screen.

### Testing instructions

* Go to the edit lesson screen.
* After the content, you should see a "Content Drip" metabox.
* Make the metabox looks good on mobile and the link points to the pricing page.
* Make sure the dropdown is working (showing the options).
* Make sure the metabox looks good when in the sidebar.

### New Hooks

* `sensei_lesson_content_drip_hide` - Filters the Content Drip toggle.

### Screenshot

Desktop:
<img width="1099" alt="Screenshot on 2022-02-12 at 04-13-34" src="https://user-images.githubusercontent.com/1612178/153692510-a187f488-4828-4e40-9a1b-080dff0dba32.png">

Mobile:
<img width="412" alt="Screenshot on 2022-02-12 at 04-12-45" src="https://user-images.githubusercontent.com/1612178/153692487-f5916999-9bc3-42a0-b056-cedb931d11a3.png">
